li-utils/src/test/java/com/linkedin/common/urn/DatasetFieldUrnTest.java

```diff
@@ -32,7 +32,7 @@ public void testSerialization() throws URISyntaxException {
   }
 
   @Test
-  public void testCreateUrn() {
+  public void testCreateUrn() throws URISyntaxException {
     final DatasetFieldUrn datasetFieldUrn = new DatasetFieldUrn(PLATFORM, DATASET_NAME, FABRIC_TYPE, FIELD_NAME);
```

Comment:
Does not seem to related to ES7 upgrade?

Reply:
+1, also these methods don't actually throw URISyntaxException? This isn't needed?

Reply:
As mentioned in the other thread with Kerem, I agree that it is better to restrict ES7 upgrade changes to this PR.